gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+            styling with little or no special behavior that must be implemented. For these cases, the bare classes
+            are themselves a documented and stable part of the RSC API, allowing them to be used even in applications
+            which do not use React. Many of these components are also exposed via React "convenience components"
+            which simply pair the class in question with the HTML tag to which it is most typically applied.
+            Convenience components are not typically documented separated and are simply mentioned on these non-React
+            pages. In some cases however, a component may have a simple form exposed as a base CSS class while
+            also having a non-trivial React component implementing a certain amount of behavior which a non-React
+            application would have to manage themselves. In these cases, the React component is documented on a separate
+            page from the base CSS classes. Live examples on these pages are often implemented in bare HTML, though
+            some are still implemented in React in order to simplify the inclusion of things such as icons.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faRulerCombined} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Complex, compound layout documentation. These pages feature combinations of RSC components that have
+            separately have their own documentation pages, but which must also be documented in unison in order to
+            understand the full effect. For instance, one such page demonstrates the use
+            of <NxCode>nx-read-only</NxCode>, <NxCode>nx-grid</NxCode>, and <NxCode>nx-tile</NxCode> together in order
+            to implement a common data display pattern.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>
+            <GalleryNavIconCSS3 />
+          </NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation of CSS-based stylesheet utilities and helpers such as CSS custom properties (aka variables)
+            and general-purpose classes which may be applied to other RSC components or HTML elements in general.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term><NxFontAwesomeIcon icon={faSass} fixedWidth /></NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Documentation of SCSS-based stylesheet utilities and helpers such as mixins. While projects are not
+            required to use SCSS in order to use RSC, any projects which do not use SCSS cannot directly use these
+            helpers.
```

Comment:
This section contains documentation of SCSS-based stylesheet utilities and helpers such as mixins. These are required by projects that use SCSS based stylesheets.